11ty/definitions/spiritual-bypassing.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Spiritual Bypassing
+slug: spiritual-bypassing
+defined: true
+excerpt: a form of appropriation where non-Black individuals use memes and imagery featuring Black people and dark-skinned emojis
+Speech: noun
+reading:
+  - text: What Is Spiritual Bypassing? - Psychology Today
+    href: https://www.psychologytoday.com/us/blog/the-empowerment-diary/201901/what-is-spiritual-bypassing
+  - text: Avoid spiritual bypassing — Anti-Racism Daily
+    href: https://www.antiracismdaily.com/archives/spiritual-bypassing-anti-racism-daily
+  - text: “Spiritual bypassing” is the “all lives matter” of the yoga world. Don't buy it
+    href: https://www.mic.com/p/spiritual-bypassing-is-the-all-lives-matter-of-the-yoga-world-dont-buy-it-27631880
+---
+
+Also referred to as toxic positivity. Spiritual bypassing is behavior that uses spiritual ideas and practices to delay or ignore acknowledging hardship. The pursuit of “good vibes only” or “oneness” can also downplay difficulties brought up by others, especially from [minoritized](/definitions/minoritised/) groups. Spiritual bypassing is present in many Western wellness and self-help practices, primarily Yoga.
+
+## Issues
+
+Spiritual bypassing avoids deeply addressing discomfort by offering surface-level platitudes, often in the form of co-opted religious, philosophical, and spiritual practices and terminology.
+
+Not addressing discomfort means an individual will not process and internalize difficult or painful events. This escapism prevents them from working through discomfort and hardship. This approach is directly at odds with Yoga’s practice of being present and working through emotional discomfort.
+
+Failing to acknowledge hardship is especially problematic in the context of minoritized groups in wellness spaces. Downplaying both invalidates their lived experiences and creates an environment that both disincentives further participation and suppresses trauma.
+
+Wellness instructors often perpetuate spiritual bypassing by modeling the behavior they see from others practitioners, notably on social media. Oftentimes they are unaware of its negative aspects and view the behavior as a deeper investment in the practice.
+
+Furthermore, co-opted language and imagery is often drawn specifically from Indian culture and Hindu faith. These concepts are often presented with no attribution and devoid of, or in conflict with the original context.
+
+## Examples
+
+An instructor saying a phrase such as, “we are all one” or “we are all human” when confronted with difficulty or hardship.
+
```

Comment:
Telling someone to "feel better" or to "get well" or to suggest Yoga or other unsolicited medical advice when not someone's explicit medical adviser. Such comments can be hurtful and damaging to mentally and physically chronically ill and disabled people who cannot and will not "get better" or "get well."

Reply:
I like this example. Do you mind if I incorporate it?

Reply:
I’d love that!

Reply:
I think in this section we need to address power and privilege. Often the people telling someone to employ "good vibes" has systemic power and privilege. So, them telling someone who experiences institutional oppression to just "have good vibes" is an abuse of their power as well, invalidating their lived experiences.

Reply:
@ericwbailey Did you all opt no to include this example? Fine if it was intentional but want to flag in case it was not. (I find it adds an important dimension of disableism myself?)